src/patternfly/components/Select/select.scss

```diff
@@ -0,0 +1,314 @@
+        border-width: var(--pf-c-select__toggle--BorderWidth) var(--pf-c-select__toggle--BorderWidth) var(--pf-c-select__toggle--expanded--BorderBottomWidth) var(--pf-c-select__toggle--BorderWidth);
+      }
+    }
+  }
+
+  .pf-c-select-toggle__arrow {
+    margin-left: var(--pf-c-select-toggle__arrow--MarginLeft);
+
+    .pf-m-top.pf-m-expanded > & {
+      transform: rotate(-180deg);
+    }
+  }
+}
+
+.pf-c-select-toggle__text {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.pf-c-select-toggle__wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: flex-start;
+
+  // Max-width is needed for ellipsing the text. This leaves enough space for the toggle caret.
+  // However, for cases where the clear icon button or a badge are added, the correct width will need to be calculated in script
+  max-width: var(--pf-c-select-toggle__wrapper--MaxWidth);
+  white-space: normal;
+
+  // Add space between children
+  > *:not(:last-child),
+  > *:not(:only-child) {
+    margin-right: var(--pf-c-select-toggle__wrapper--not-last-child--MarginRight);
+  }
+
+  // need to reduce the padding, which is driving the height of the chip, so it fits within the toggle
+  .pf-c-chip .pf-c-button {
+    padding-top: var(--pf-c-select-toggle__wrapper--c-chip--c-button--PaddingTop);
+    padding-bottom: var(--pf-c-select-toggle__wrapper--c-chip--c-button--PaddingBottom);
+  }
+}
+
+.pf-c-select-toggle__clear {
+  padding-right: var(--pf-c-select-toggle__clear--PaddingRight);
+}
+
+.pf-c-select-toggle__badge {
+  display: flex;
+  padding-left: var(--pf-c-select-toggle__badge--PaddingLeft);
+}
+
+.pf-c-select-toggle__typeahead {
+  flex-basis: var(--pf-c-select-toggle__typeahead--FlexBasis);
+  flex-grow: 1;
+  flex-shrink: 0;
+  background-color: var(--pf-c-select-toggle__typeahead--BackgroundColor);
+  border-top: var(--pf-c-select-toggle__typeahead--BorderTop);
+  border-right: var(--pf-c-select-toggle__typeahead--BorderRight);
+  border-bottom-color: inherit;
+  border-bottom-style: inherit;
+  border-left: var(--pf-c-select-toggle__typeahead--BorderLeft);
+}
+
+.pf-c-select__menu {
+  position: absolute;
+  top: var(--pf-c-select__menu--Top);
+  z-index: var(--pf-c-select__menu--ZIndex);
+  min-width: 100%;
+  padding-top: var(--pf-c-select__menu--PaddingTop);
+  padding-bottom: var(--pf-c-select__menu--PaddingBottom);
+  background: var(--pf-c-select__menu--BackgroundColor);
+  background-clip: padding-box;
+  border: var(--pf-c-select__menu--BorderWidth) solid transparent;
+  box-shadow: var(--pf-c-select__menu--BoxShadow);
+
+  &.pf-m-align-right {
+    right: 0;
+  }
+
+  .pf-c-select.pf-m-top & {
+    top: 0;
+    transform: var(--pf-c-select__menu--top--Transform);
+  }
+}
+
+.pf-c-select__menu-item {
+  display: block;
```

Comment:
this breaks `display: grid` when used on `.pf-c-check`

Reply:
I will remove it; doesn't seem to harm anything to be removed - but this was copied from dropdown, where it still exists. I'm not sure if it's necessary/harmful there.